README.md

```diff
@@ -1,10 +1,278 @@
 # CactusBot
 
-## Open Source and Community Driven
+CactusBot is a next-generation ChatBot for LiveStreams.
+Harnessing the power of open-source, and extraordinary community to shape its path
 
-Including stunning, innovative functionalities, CactusBot is an entirely
-next-generation chat bot, ultimately harnessing the power of open source
-alongside an amazing community to shape its path.
+![screenshot of the bot running in a terminal](./assets/terminal.png)
 
-### Build Status
-[![Build Status](https://travis-ci.org/CactusDev/CactusBot.svg?branch=master)](https://travis-ci.org/CactusDev/CactusBot)
+# Installation
+
+## LINUX:
+***PYTHON 3.5 OR GREATER IS REQUIRED. ANY VERSION LOWER IS NOT SUPPORTED***
```

Comment:
It's important to make that stand out. I don't ever want to see an issue about it not working because they're on 3.4 or less and it's erroring on `async` or `await`.